db/migrate/20170508150135_add_foreign_keys.rb

```diff
@@ -0,0 +1,49 @@
+class AddForeignKeys < ActiveRecord::Migration
+  def change
+    add_index :notifications, [:subject_id, :subject_type]
+    add_index :order_details, :project_id
+    add_index :order_statuses, :parent_id
+    add_index :orders, :merge_with_order_id
+    add_index :price_group_members, :account_id
+    add_index :products, :initial_order_status_id
```

Comment:
Am I correct in assuming that any other foreign keys we're adding already had indexes?